drivers/serial/uart_native_posix.c

```diff
@@ -41,11 +41,22 @@ static bool auto_attach;
 static const char default_cmd[] = CONFIG_NATIVE_UART_AUTOATTACH_DEFAULT_CMD;
 static char *auto_attach_cmd;
 
+static struct native_uart_status native_uart_status_0;
```

Comment:
Nit: I guess the explanation in the top would need a minor update (?):
```diff
diff --git a/drivers/serial/uart_native_posix.c b/drivers/serial/uart_native_posix.c
index 75e53f5f2f..10aeef611a 100644
--- a/drivers/serial/uart_native_posix.c
+++ b/drivers/serial/uart_native_posix.c
@@ -20,6 +20,10 @@
 
 /*
  * UART driver for POSIX ARCH based boards.
+ * It can support up to two UARTs.
+ *
+ * For the first UART:
+ *
  * It can either be connected to the process STDIN+STDOUT
  * OR
  * to a dedicated pseudo terminal
```